accounts/admin.py

```diff
@@ -1,3 +1,35 @@
 from django.contrib import admin  # noqa
 
 # Register your models here.
+from accounts.models import User
+
+
+class UserAdmin(admin.ModelAdmin):
+    list_display = [
+        "email",
+        "first_name",
+        "last_name",
+        "gender",
+        "role",
+        "is_active",
+        "is_staff",
+        "is_superuser",
+        "date_joined",
+        "last_login",
+    ]
+    list_filter = ["role", "gender", "is_active", "is_staff", "is_superuser"]
+    fields = [
```

Comment:
Mmm... I think `fields` is not required. By default, all fields are showed.

Reply:
It's not required, but password, user_permissions and groups are not used, so I think that can be excluded.

Using this can't fix the non excluded fields order.
```bash
exclude = ['password', 'groups', 'user_permissions']
```